File: hw1/python/person.py
# Jake Felzien
# moving classes to other files
class Person():
    def __init__(self, BirthCounty, BirthDay, BirthMonth, BirthOrder, BirthYear, FirstName, Gender, IsPartOfMultipleBirth, LastName, MiddleName, MotherFirstName, MotherLastName, MotherMiddleName, NewbornScreeningNumber, ObjectId, Phone1, Phone2, SocialSecurityNumber, StateFileNumber):
        self.BirthCounty = BirthCounty
        self.BirthDay = BirthDay
        self.BirthMonth = BirthMonth 
        self.BirthOrder = BirthOrder
        self.BirthYear = BirthYear
        self.FirstName = FirstName
        self.Gender = Gender
        self.IsPartOfMultipleBirth = IsPartOfMultipleBirth
        self.LastName = LastName
        self.MiddleName = MiddleName
        self.MotherFirstName = MotherFirstName
        self.MotherLastName = MotherLastName
        self.MotherMiddleName = MotherMiddleName
        self.NewbornScreeningNumber = NewbornScreeningNumber
        self.ObjectId = ObjectId
        self.Phone1 = Phone1
        self.Phone2 = Phone2
        self.SocialSecurityNumber = SocialSecurityNumber
        self.StateFileNumber = StateFileNumber
    def printPerson(self):
        stringer = "\t"+str(self.ObjectId)+" "+ str(self.FirstName) + " " +str(self.MiddleName)+ " " + str(self.LastName) + " " + str(self.BirthDay)+"/"+str(self.BirthMonth) + "/"+ str(self.BirthYear)
        print stringer

File: hw1/c#/strategyPattern/MyClasses/Person.cs
using System.Collections.Generic;
using System.IO;
using System.Runtime.Serialization.Json;

namespace MyClasses
{
    public class Person
    {
        public string BirthCounty{get; set;}
        public string BirthDay {get; set;}
        public string BirthMonth {get; set;}
        public string BirthOrder {get; set;}
        public string BirthYear {get; set;}
        public string FirstName {get; set;}
        public string Gender {get; set;}
        public string IsPartOfMultipleBirth {get; set;}
        public string LastName {get; set;}
        public string MiddleName {get; set;}
        public string MotherFirstName {get; set;}
        public string MotherLastName {get; set;}
        public string MotherMiddleName {get; set;}
        public string NewbornScreeningNumber {get; set;}
        public string ObjectId {get; set;}
        public string Phone1 {get; set;}
        public string Phone2 {get; set;}
        public string SocialSecurityNumber {get; set;}
        public string StateFileNumber {get; set;}        
    
    public override string ToString(){
        return "Id=" + ObjectId + ", Name="+FirstName+ MiddleName+ LastName + ", Birthday="+BirthDay+"/"+BirthMonth+"/"+BirthYear;
    }

    }
}
